nss-cohort-36/bangazon-workforce-mgt-virtual-magicians
# Cb details

# Description
When viewing http://localhost:8000/computers/ the user should see a list of clickable computers. Clicking a computer link should lead to the details of that computer.
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. Ensure that connection.py contains the appropriate absolute path for your machine to test locally
1. `git checkout master`
1. `git fetch --all`
1. `git checkout cb-details`
1. Go to http://localhost:8000/computers/ and try clicking on a link. Please confirm that the computer details are being displayed.
# Checklist:
